Sample Applications/CustomComboBox/MainWindow.xaml.cs

```diff
@@ -28,5 +29,14 @@ public MainWindow()
         private void Button_Click(object sender, RoutedEventArgs e)
         {
         }
+
+        private void TextBlock_TargetUpdated(object sender, DataTransferEventArgs e)
+        {
+            var listingPeer = ListBoxAutomationPeer.FromElement(sender as TextBlock);
```

Comment:
Check that the event is relevant before doing this work.
``` if (e.Property == TextBlock.TextProperty) { ... } ```

Reply:
Hi Sam, 
I did the changes you requested.